flink-core/src/main/java/org/apache/flink/api/common/PlanExecutor.java

```diff
@@ -129,16 +110,16 @@ public static PlanExecutor createRemoteExecutor(
 		}
 	}
 
-	private static Class<? extends PlanExecutor> loadExecutorClass(String className) {
```

Comment:
I don't think we should modify this method in this pass. `loadExecutorClass` implicitly load remote executor makes non-sense. We can remote the whole `PlanExecutor` when we later get rid of `RemoteExecutor extends PlanExecutor`.